ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragment/v1/fragment/fragmentplaceholder.html

```diff
@@ -0,0 +1,26 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can we rename this file to `placeholder.html`, since we are already inside fragment context

Reply:
ok